master/reference/policy-controller/configuration.md

```diff
@@ -13,57 +13,35 @@ section.
 The policy controller supports the following environment variables to configure
 etcd access:
 
-* `ETCD_ENDPOINTS`: The list of etcd nodes in your cluster. e.g `http://10.0.0.1:2379,http://10.0.0.2:2379`
-* `ETCD_CA_CERT_FILE`: The full path to the CA certificate file for the Certificate Authority that signed the etcd server key/certificate pair.
-* `ETCD_CERT_FILE`: The full path to the client certificate file for accessing the etcd cluster.
-* `ETCD_KEY_FILE`: The full path to the client key file for accessing the etcd cluster.
+| Environment   | Description | Schema |
+| ------------- | ----------- | ------ |
+| ETCD_ENDPOINTS    | The list of etcd nodes in your cluster. e.g `http://10.0.0.1:2379,http://10.0.0.2:2379`
+| ETCD_CA_CERT_FILE | The full path to the CA certificate file for the Certificate Authority that signed the etcd server key/certificate pair. | path
+| ETCD_CERT_FILE    | The full path to the client certificate file for accessing the etcd cluster. | path
+| ETCD_KEY_FILE     | The full path to the client key file for accessing the etcd cluster. | path
 
-> **Note**: When running etcd with TLS enabled, the addresses in 
-> `ETCD_ENDPOINTS` must be hostname values such as `etcd-host:2379`, 
-> NOT IP addresses.
-{: .alert .alert-info}
+The `*_FILE` variables are _paths_ to the corresponding certificates / keys. As such, when the policy controller is running as a Kubernetes pod, you
```

Comment:
I think that the additional spaces around the `/` are not necessary. I would delete those so that it read `certificates/keys`.